docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hollweg Dispersion Solver\n",
+    "\n",
+    "[tfds]: ../../api/plasmapy.dispersion.numerical.hollweg_.hollweg.rst\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "[hollweg1999]: https://doi.org/10.1029/1998JA900132\n",
+    "\n",
+    "This notebook details the functionality of the [hollweg()][tfds] function.\n",
+    "This function computes the wave frequencies for given wavenumbers and\n",
+    "plasma parameters based on the solution presented by [Bellan 2012][bellan2012]\n",
+    "to the [Hollweg 1999][hollweg1999] two fluid dispersion relation.\n",
+    "The dispersion relation assumes a uniform magnetic field, zero D.C\n",
+    "electric field, quasi-neutrality, wave propogation nearly perpendicular to\n",
+    "the magnetic field, a low-beta regime $c_{s} \\ll v_{A}$,\n",
+    "and low-frequency waves $\\omega \\ll \\omega_{ci}$ which equates to\n",
+    "\n",
+    "$$\n",
+    "    \\left( \\frac{\\omega^2}{{k_z}^2 {v_A}^2} - 1 \\right)\n",
+    "    \\left[\\omega^2\n",
+    "        \\left(\\omega^2 - k^2 {v_A}^2 \\right)\n",
+    "        - \\beta k^2 {v_A}^2 \\left( \\omega^2 - {k_z}^2 {v_A}^2 \\right)\n",
+    "    \\right]\n",
```

Comment:
```suggestion
    "    \\right] \\\\\n",
```

Add an explicit line return for the equation so the equation does not run off the screen in the RTD builds.